crates/uv-pep508/src/marker/algebra.rs

```diff
@@ -271,10 +271,53 @@ impl InternerGuard<'_> {
                 key,
                 operator,
                 value,
-            } => (
-                Variable::String(key.into()),
-                Edges::from_string(operator, value),
-            ),
+            } => {
+                // Normalize `platform_system` markers to `sys_platform` nodes.
```

Comment:
I would feel more confident if we could codify this upstream (packaging.python.org probably), so uv doesn't end up making assumptions that python redistributors don't make.

Reply:
Do you want to open a convo about it?

Reply:
Also, do we have any context on _why_ there is `platform_system` and `sys_platform`?

Reply:
And maybe it's obvious, but I think a comment saying why we normalize to `sys_platform` (instead of the other way around) would be good too.

Reply:
https://discuss.python.org/t/clarify-usage-of-platform-system/70900